CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ This is the log of notable changes to the Expo client that are developer-facing.
 
 - Added `fallbackLabel` option to `LocalAuthentication.authenticateAsync` on iOS which allows to customize a title of the fallback button when the system doesn't recognize the user and asks to authenticate via device passcode. ([#4612](https://github.com/expo/expo/pull/4612) by [@changLiuUNSW](https://github.com/changLiuUNSW))
 - added `native` mode for Android SplashScreen on standalone apps by [@bbarthec](https://github.com/bbarthec) ([#4567](https://github.com/expo/expo/pull/4567))
+- added support for video recording in `ImagePicker.launchCameraAsync`. ([#4903](https://github.com/expo/expo/pull/4903) by [@lukmccall](https://github.com/lukmccall))
```

Comment:
nitpicking, by convention is to first describing `by [@who]` part and then PR reference 😉

Reply:
You're not right @bbarthec 😛 That's how we did, but now I'm encouraging to use `(#4903 by @lukmccall) instead. This way it is more clear and the fact who did the change isn't as important as the link to the pull request where people can find more info.